profiler/src/ProfilerEngine/Datadog.Profiler.Native/ErrorCode.h

```diff
@@ -0,0 +1,35 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+namespace libdatadog {
+
+namespace detail {
+    struct ErrorCodeImpl;
+}
+
+class ErrorCode
```

Comment:
This class should be named Success and the ErrorCodeImpl might not be needed if the "error" case is identified (i.e. FfiHelper::ExtractMessage(the wrapped ddog_Error)) to extract the string message and call ddog_Error_drop immediately.

Reply:
✔️